diff --git a/build.gradle b/build.gradle
index b2005e933..32e1e200f 100644
--- a/build.gradle
+++ b/build.gradle
@@ -12,6 +12,9 @@ buildscript {
             url 'https://oss.sonatype.org/content/groups/public'
         }
     }
+    dependencies {
+         classpath "org.qmstr:gradle-plugin:0.3.0"
+     }
 }
 
 plugins {
@@ -39,6 +42,8 @@ apply plugin: 'install4j'
 apply plugin: 'me.champeau.gradle.jmh'
 apply plugin: 'checkstyle'
 
+apply plugin: 'qmstr'
+
 apply from: 'eclipse.gradle'
 apply from: 'localization.gradle'
 apply from: 'xjc.gradle'
