From 01fbc4262df9cc0569fe6c144bae059bda194d9a Mon Sep 17 00:00:00 2001
From: Giasemi Seisa <giasemi.seisa@endocode.com>
Date: Mon, 16 Apr 2018 14:27:56 +0200
Subject: [PATCH] [PATCH] Add qmstr plugin

---
 build.gradle | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/build.gradle b/build.gradle
index 28061e1b2..a93dcdb9c 100644
--- a/build.gradle
+++ b/build.gradle
@@ -14,6 +14,7 @@ buildscript {
     }
     dependencies {
         classpath 'org.junit.platform:junit-platform-gradle-plugin:1.2.0-M1'
+        classpath "org.qmstr.gradle:qmstr-plugin:0.1"
     }
 }
 
@@ -43,6 +44,8 @@ apply plugin: 'me.champeau.gradle.jmh'
 apply plugin: 'checkstyle'
 apply plugin: 'org.junit.platform.gradle.plugin'
 
+apply plugin: 'qmstr'
+
 apply from: 'eclipse.gradle'
 apply from: 'localization.gradle'
 apply from: 'xjc.gradle'
-- 
2.16.3

